websites/O/OneTouchTV/metadata.json

```
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "638080361179512853",
+		"name": "dark_ville"
+	},
+	"service": "OneTouchTV",
+	"description": {
+		"en": "OneTouch TV is the hit Drama & Movie & TV app . OneTouch TV has it all in a single app. You can stream all your favorite movies & drama & TV for free."
+	},
+	"url": "onetouchtv.me",
```

The domain appears to have changed. Perhaps a regex may be useful as well?
```suggestion
	"url": "onetouchtv.cc",
```